spring-boot-project/spring-boot-devtools/src/main/java/org/springframework/boot/devtools/livereload/Connection.java

```diff
@@ -106,7 +108,7 @@ private void readWebSocketFrame() throws IOException {
 			else if (frame.getType() == Frame.Type.CLOSE) {
 				throw new ConnectionClosedException();
 			}
-			else if (frame.getType() == Frame.Type.TEXT) {
+			else if (frame.getType() == Frame.Type.TEXT && logger.isDebugEnabled()) {
```

Comment:
This will cause `TEXT` frames to be unexpected when debug logging isn't available.

Reply:
I will modify it later, thank you for your reply.